superset/cachekeys/schemas.py

```diff
@@ -51,3 +51,44 @@ class CacheInvalidationRequestSchema(Schema):
         fields.Nested(Datasource),
         metadata={"description": "A list of the data source and database names"},
     )
+
+
+class CacheWarmUpRequestSchema(Schema):
+    chart_id = fields.Integer(
+        metadata={"description": "The ID of the chart to warm up cache for"}
+    )
+    dashboard_id = fields.Integer(
+        metadata={
+            "description": "The ID of the dashboard to get filters for when warming cache"
+        }
+    )
+    table_name = fields.String(
```

Comment:
The main reason is that the old endpoint used the table name, and it didn't occur to me to change it.  I think it makes sense to keep it though since it makes it much easier for anyone using the old endpoint currently to migrate to the new one.  Also table names are unique within a database, so I think it's safe to use as an easier-to-read identifier.